assets/src/edit-story/components/canvas/layout.js

```diff
@@ -102,6 +102,21 @@ const PageAreaFullbleedContainer = styled(Area).attrs({
   display: flex;
   justify-content: center;
   align-items: center;
+
+  ${({ isBackgroundSelected, theme }) =>
+    isBackgroundSelected &&
+    `
+    &:before {
+      content: '';
+      position: absolute;
+      top: -4px;
+      left: -4px;
+      right: -4px;
+      bottom: -4px;
+      border: ${theme.designSystemTheme.colors.blue[20]} 1px solid;
+      border-radius: 10px;
```

Comment:
The radius-vs-distance looks off.

Compare:

| 4px + 10px | 4px + 8px | 5px + 10px |
|-|-|-|
| ![Screen Shot 2021-02-08 at 17 16 54](https://user-images.githubusercontent.com/637548/107288775-fe75b200-6a31-11eb-8543-4bf843870769.png) | ![Screen Shot 2021-02-08 at 17 17 08](https://user-images.githubusercontent.com/637548/107288773-fddd1b80-6a31-11eb-8606-af0be1291da3.png) | ![Screen Shot 2021-02-08 at 17 19 17](https://user-images.githubusercontent.com/637548/107288772-fddd1b80-6a31-11eb-933a-055cda626d36.png) |

I don't know which of the latter two is the right one, but the first one definitely seems off.

Reply:
Good call, Not sure either, went with 5/10 for now.